content/en/docs/armory-agent/armory-agent-quick.md

```diff
@@ -5,48 +5,146 @@ description: >
   Learn how to install the Armory Agent in your Kubernetes and Armory Enterprise environments.
 weight: 2
 ---
-![Proprietary](/images/proprietary.svg)
+![Proprietary](/images/proprietary.svg) This feature requires an Armory licensed entitlement.
 
-## Compatibility matrix
+## Overview
 
-{{< include "agent/agent-compat-matrix.md" >}}
-
-The Agent consists of a service deployed as a Kubernetes `Deployment` and a plugin to Spinnaker's Clouddriver service. You can review the architecture in the Armory Agent [overview]({{< ref "armory-agent" >}}).
+The Agent consists of a plugin to Spinnaker's Clouddriver service and a K8s deployment that connects to Clouddriver. You can review the architecture in the Armory Agent [overview]({{< ref "armory-agent" >}}).
 
 ## {{% heading "prereq" %}}
 
-* This guide is for experienced Kubernetes and Armory Enterprise users.
+* This guide assumes you're using the Armory Operator to install Spinnaker, with the Kustomize method from the [spinnaker-kustomize-patch repo](https://github.com/armory/spinnaker-kustomize-patches).
+
 * You have read the Armory Agent [overview]({{< ref "armory-agent" >}}).
-* You have a Redis instance. The Agent uses Redis to coordinate between Clouddriver replicas.
```

Comment:
Why is Redis removed? It's a requirement if you operate more than one clouddriver.

Reply:
I think it could be moved to the main doc, to keep the quick start more streamlined and simple.